source/js/navigation.js

```diff
@@ -132,6 +153,10 @@ $(document).ready(function () {
         return (url_of_img.replace(image_name, name_of_file_no_extension + '.png'));
     }
 
+    function urlContains(string) {
+        return (document.location.pathname.indexOf(string) > -1) ? true : false;
```

Comment:
as discussed elsewhere, `x ? true : false` can be replaced with `!!x` or simply `x` if x is already boolean